aspen/resources/__init__.py

```diff
@@ -97,6 +97,52 @@ def get_declaration(line):
     return fulltext.decode(encoding or b'ascii')
 
 
+
+class SimplateWrapper(Simplate):
```

Comment:
I think of `SimplateWrapper` as an Aspen-specific wrapper for simplates.  It can potentially be refactored away so that 'raw' simplates could be used instead, but that'd require more massive changes than I was aiming for.